masterlist.yaml

```diff
@@ -401,6 +404,12 @@ plugins:
       - 'SoS - The Wilds.esp'
       - 'SoS - The Dungeons.esp'
       - 'SoS - Civilization.esp'
+    tag:
+      - C.Location
+      - C.Owner
+      - C.Regions
+      - C.Water
+      - Graphics
```

Comment:
I haven't looked at Realistic Water Two in a while, but are C.Location and C.Owner really appropriate?

Reply:
Location yes, but now that you mention it I want to look at C. Owner.

Reply:
Actually @WrinklyNinja since Irma decided she wanted to visit Space Camp after visiting Disney World, I may not get to look at this tonight, especially if we don't have power.

I've edited my fork so I'm going to close this PR and submit a fresh one.

Reply:
OK it's raining sideways here but I figured out the C.Owner tag on RWT: the plugin sets player ownership of a RWT-placed horse trough (RWT_HorseTrough) in the various Hearthfires homes. 

It'll probably be fine to leave it out. If we ever hear of anything clobbering the RWT Horse Troughs we can add it back.